CacheAdvance.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
```

Comment:
Starting a thread for this discussion:

> My concern is there is still this line of code: [`try reader.seek(to: FileHeader.expectedEndOfHeaderInFile)` ](https://github.com/dfed/CacheAdvance/pull/66/files#diff-90b1f5431bb05caf1445ee7ee42e2a66b2bfb2e6fad63f3266cace6cd1389fd6L68), which keeps the possibility to run into an infinite loop. 

I don't think that line can lead to an infinite loop per the unit tests in #66. We will either read all messages and then hit the end and stop, or we will detect that the file is corrupted and stop.

That said, I think the approach you're taking here may be easier to maintain + understand than the approach I took in #66, so I'm open to this approach even if we're not solving an existing problem. However, I would feel much more confident with this approach if we didn't revert the tests introduced in #66.

Reply:
That said, this new code may have different behavior (maybe we don't throw `fileCorrupted` anymore?), so we may need to _update_ the tests from #66.

Reply:
> However, I would feel much more confident with this approach if we didn't revert the tests introduced in https://github.com/dfed/CacheAdvance/pull/66.

That's helpful. Let me keep the tests.

Reply:
Per https://github.com/dfed/CacheAdvance/pull/72#discussion_r1013594675, it sounds like we're intending to keep the tests from #66! Excited to see if they need to be modified to pass with this change 🙂